hack/featuregate_backup.yaml

```diff
@@ -0,0 +1,14 @@
+---
+apiVersion: config.openshift.io/v1
+kind: FeatureGate
+metadata:
+  annotations:
+    include.release.openshift.io/self-managed-high-availability: "true"
+    include.release.openshift.io/single-node-developer: "true"
+    release.openshift.io/create-only: "true"
+  name: cluster
+spec:
+  customNoUpgrade:
+    enabled:
+      - EtcdBackupAPI
+  featureSet: CustomNoUpgrade
```

Comment:
Was this just for testing or is this where we read the feature flag?

Reply:
nope that's just for easier testing against cluster bot, to enable the feature so to say